src/DGmethods/DGmodel.jl

```diff
@@ -46,6 +46,7 @@ function (dg::DGModel)(dQdt, Q, param, t; increment=false)
   # Gradient Computation #
   ########################
   MPIStateArrays.start_ghost_exchange!(Q)
+  MPIStateArrays.start_ghost_exchange!(auxstate)
```

Comment:
@lcw @jkozdon We update the aux in each iteration: is this the best way to do this?

Reply:
For now this is the best we can do. Another long term option would be to pack on to the end of `Q` any `auxstate` that will change in time. This would mean flagging in the ODE solver what fields are and are not update by the ODE solver.

Reply:
The other option is to update aux both before and after communicating Q. You should be able to update aux on the ghost element from the ghost Q, this would fix the whole issue. To do this you'd need to call `update_aux!` on the ghost elements after finishing the communication of Q

Reply:
If it stays as is, on line 80 you should add `nviscstate == 0 && MPIStateArrays.finish_ghost_recv!(auxstate)`